frontend/src/slices/meditation/meditation.slice.ts

```diff
@@ -0,0 +1,37 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+import { DataStatus } from '#libs/enums/enums.js';
+import { type ValueOf } from '#libs/types/types.js';
+import { type MeditationEntryCreateResponseDto } from '#packages/meditation/meditation.js';
+
+import { createMeditationEntry } from './actions.js';
+
+type State = {
+  meditationEntries: MeditationEntryCreateResponseDto[];
+  dataStatus: ValueOf<typeof DataStatus>;
```

Comment:
What if we would need separate data status for creating an entry or anything else, like in other slices?
```suggestion
  meditationEntriesDataStatus: ValueOf<typeof DataStatus>;
```

Reply:
Done